index.html

```diff
@@ -39,6 +39,7 @@
 
 <p>If you would like to remove personal information from a bug, please <a href="https://github.com/xamarin/bugzilla-archives/issues/new/choose">file an issue</a>.
     
+<p>For internal bugs not immediately accessible, please contact the Release Engineering team.</p> 
```

Comment:
I'd move this above the removing personal info notice and reword slightly:

```suggestion
<p><b>Microsoft/Xamarin employees</b>: To access private bugs please contact the Release Engineering team.</p>

<p>If you would like to remove personal information from a bug, please <a href="https://github.com/xamarin/bugzilla-archives/issues/new/choose">file an issue</a>.

```